test/SearchCriteriaTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Acquia\ContentHubClient\test;
+
+use PHPUnit\Framework\TestCase;
+use Acquia\ContentHubClient\SearchCriteria\Transformer;
+
+
+class SearchCriteriaTest extends TestCase {
+
+  /**
+   * @var array
+   */
+  private $data;
+
+  /**
+   * @var array
+   */
+  private $expected;
+
+  public function setUp() {
+    $this->data = [
+      'search_term' => 'some-search-term',
+      'type' => 'some-type',
+      'bundle' => 'some-bundle',
+      'tags' => 'some-tags',
+      'label' => 'some-label',
+      'start_date' => 'some-start_date',
+      'end_date' => 'some-end-date',
+      'from' => 'some-from',
+      'size' => 'some-size',
+      'sorting' => 'some-sort',
+      'version' => 'some-version',
+      'languages' => [
+        'some-language-1',
+        'some-language-2',
+      ],
+    ];
+
+    $this->expected = [
+      'search_term' => 'some-search-term',
+      'type' =>
+        [
+          0 => 'some-type',
+        ],
+      'bundle' =>
+        [
+          0 => 'some-bundle',
+        ],
+      'tags' =>
+        [
+          0 => 'some-tags',
+        ],
+      'label' => 'some-label',
+      'start_date' => 'some-start_date',
+      'end_date' => 'some-end-date',
+      'from' => 'some-from',
+      'size' => 'some-size',
+      'sorting' => '',
+      'version' => 'some-version',
+      'languages' =>
+        [
+          0 => 'some-language-1',
+          1 => 'some-language-2',
+        ],
+    ];
+  }
+  public function test_result_when_all_keys_are_present_in_data(): void {
+    $data = $this->data;
+    $expected = $this->expected;
+
+    self::assertEquals(Transformer::arrayToSearchCriteriaArray($data), $expected);
+  }
+
+  public function test_missing_keys_get_default_values(): void {
```

Comment:
Why does this differ in terms of code style from the other tests?

Reply:
Could you please elaborate on this?

Reply:
I mean here you use snake cased method names, while in other test classes camel case is used.